package-lock.json

```diff
@@ -16,7 +16,7 @@
 		"@babel/helper-function-name": {
 			"version": "7.0.0-beta.31",
 			"resolved": "https://registry.npmjs.org/@babel/helper-function-name/-/helper-function-name-7.0.0-beta.31.tgz",
-			"integrity": "sha512-c+DAyp8LMm2nzSs2uXEuxp4LYGSUYEyHtU3fU57avFChjsnTmmpWmXj2dv0yUxHTEydgVAv5fIzA+4KJwoqWDA==",
+			"integrity": "sha1-r+Y615kgmYk0ixEJtE/rZqokX1c=",
```

Comment:
I see we're doing a bit of a dance on hash algorithms. In my case, running `npm install` on a fresh clone results in a changed `package-lock.json` flipping back to `sha512`.

I'm not sure what causes this: Something about how you updated the package? OS? Version of Node/npm ?

Reply:
Hrrm, that's interesting. I manually updated the version in `package.json`, then ran `npm install`.

Running MacOS 10.13.4, Node 8.11.1, npm 5.6.0.

Reply:
I'm running largely the same, save for npm 5.8.0 . I asked @youknowriad and he mentioned he'd seen it once, but not consistently. Let's keep an eye on it.

Reply:
I've had this issue numerous times, I contacted npm about a similar issue ~6 months ago, the solution was to empty the cache and start again:

> `npm cache clear -f && rm -rf package-lock.json node_modules && npm install`

In particular, clearing the cache is what got it sorted

Reply:
@ntwb By "got it sorted", should we want sha512 to be the committed default then?

Reply:
@aduth Preferably, for the past x months, its been a _ crapshoot_, if I got `sha512`'s then 🎉, if not then I'd do whatever I could to minimise changes to the lock file, including but not limited to manually editing the lock file or simply giving up and trying again another day /shrug